velox/exec/tests/utils/PlanBuilder.cpp

```diff
@@ -1227,8 +1227,10 @@ PlanBuilder& PlanBuilder::window(
   };
 
   WindowTypeResolver windowResolver;
+  facebook::velox::duckdb::ParseOptions options;
+  options.parseIntegerAsBigint = options_.parseIntegerAsBigint;
   for (const auto& windowString : windowFunctions) {
-    const auto& windowExpr = duckdb::parseWindowExpr(windowString);
+    const auto& windowExpr = duckdb::parseWindowExpr(windowString, options);
```

Comment:
Why not pass options_ here instead of creating the new local options?

Reply:
options_ is instance of `parse::ParseOptions`, but we need `duckdb::ParseOptions` here, so I create a local `options `here.